app/resources/translations/en_GB/messages.en_GB.yml

```diff
@@ -121,14 +121,14 @@ field.image.label.alt: "Alt"
 field.markdown.label.markedview: "Markdown"
 field.markdown.label.preview: "Preview"
 
-field.oembed.height: "Height"
-field.oembed.label.preview: "Preview"
-field.oembed.label.size: "Size"
-field.oembed.label.url: "URL of content to embed"
-field.oembed.matched-oembed: "Matched Embed"
-field.oembed.pixel: "pixel"
-field.oembed.placeholder.url: "URL of content on Facebook, Twitter, Soundcloud or other media website"
-field.oembed.width: "Width"
+field.embed.height: "Height"
+field.embed.label.preview: "Preview"
+field.embed.label.size: "Size"
+field.embed.label.url: "URL of content to embed"
+field.embed.matched-oembed: "Matched Embed"
```

Comment:
`matched-oembed` or `matched-video`?
`field.embed.matched-oembed: "Matched Embed"`

nl-NL was changed to:
`field.embed.matched-video: "Gevonden embed"`

Reply:
Nice catch, thank you :+1:

Reply:
Addressed in 94008726ba004c760b5d07204074eecb4b38b5df 

Matching use is: https://github.com/bolt/bolt/blob/f987db921007f13ac2077bd62859b0fea3384823/app/view/twig/editcontent/fields/_embed.twig#L144